subprojects/persistent-cache/src/main/java/org/gradle/cache/internal/AbstractCacheCleanup.java

```diff
@@ -40,7 +40,7 @@ public void clean(CleanableStore cleanableStore, CountdownTimer timer) {
         int filesDeleted = 0;
         for (File file : findEligibleFiles(cleanableStore)) {
             if (timer.hasExpired()) {
-                LOGGER.warn("{} cleanup was aborted because timeout has expired", cleanableStore.getDisplayName());
+                LOGGER.info("{} cleanup was aborted because timeout has expired", cleanableStore.getDisplayName());
```

Comment:
What about making it explicit that it's going to retry?

> cleanup was aborted because it took too long to complete. Cleanup will however be resumed later.

Reply:
Very good point. Changed in d994da4.